src/config/config.model.ts

```diff
@@ -50,11 +50,14 @@ export const emptyConfig: Config = {
 };
 
 function getConfigPath(): string {
+  const path = join(".config", "collie-cli");
+  var home = "";
```

Comment:
let instead of var is preferred in newer JS/TS versions. var behaves very weird:
```
if(true) {
var a = "bla";
}

console.log(a); // works as a now lives in the whole scope.
```

```
if(true) {
let a = "bla";
}

console.log(a); // fails, let limits variables scope to the enclosing brackets.
```
Funnily you could place `var home = '';` inside the ifs and it would still work ;)

Reply:
fixed

Reply:
I'm also a bit confused why `deno lint` doesn't trigger on `var`. Sad that we cannot modify the linting config :/